modules/learn/pages/services-and-indexes/indexes/index-lifecycle.adoc

```diff
@@ -0,0 +1,53 @@
+= Index Lifecycle
+:page-topic-type: concept
+:imagesdir: ../../../assets/images
+:description: An overview of the lifecycle of a Global Secondary Index, from creation and building to updates and scans.
+
+[abstract]
+{description}
+
+In Couchbase Server 7.0 and later, a Global Secondary Index is created on a single collection -- not on an entire bucket.
```

Comment:
Index lifecycle page should include the meaning of index creation and index build. The details mentioned in modules/n1ql/pages/n1ql-intro/queriesandresults.adoc can be used here as well.

Reply:
Done